### ISSUE TITLE
Question on how to use this for inference

### ISSUE THREAD

**Issue by Husky95** [icoz69/DeepEMD#47]
So i'm able to train my own model on a custom dataset, but had a hard time figuring own how to do inference on this. For example given a single image, how do I make it so that the result is the prediction of what class the image belong to and the confidence scores of the prediction .

**Comment by icoz69** [comment]
hello, i would suggest you go through the eval.py and replace the dataloader with your own dataset. the output from the network is the predicted logits, and you can use argmix to obtain the predicted class.
